### ISSUE TITLE
enemies not activated

### ISSUE THREAD

**Issue by x3003** [Realm667/WolfenDoom#1461]
In c2m6_a, these enemies are not activated until they see you in a direct line.
Normally they would see you through the glass and get activated

![Screenshot_Doom_20240915_004158](https://github.com/user-attachments/assets/6e39fdec-da5e-46dc-8987-87f65514093d)
![Screenshot_Doom_20240915_004256](https://github.com/user-attachments/assets/a2e24720-13cd-4d94-a245-645400358fba)

**Comment by hawkwind3** [comment]
Not a bug. Lines 12134, 12131,12132 and 12133, which are  near the "glass", are flagged "block everything", which included line-of-sight. Same with BoA 3.1.

**Comment by x3003** [comment]
Strange, they always came for me when walking up the stairs,,,

**Comment by hawkwind3** [comment]
You are correct in saying that there is now a difference. My apologies. And ignore what i stated about those 4 lines. It has nothing to do with it.
With BoA 3.1, all the enemies around the shark tank are alerted if you fire your weapon at the bottom of the stairs. They are not in BoA 3.2 beta. I'm going to hazard a guess though, in saying that BoA3.1 is incorrect, because all those stairs are flagged "block sound" in both 3.1 and 3.2.
I'm no expert, so left open for further comment by anyone.
